scorecard/profile_data.py

```diff
@@ -249,6 +254,44 @@ def revenue_breakdown(self):
                 continue
         return {'values': values}
 
+    def current_ratio(self):
+        values = []
+        results = self.results['in_year_bsheet']
+        year_key = lambda r: r['financial_year_end.year']
+        month_key = lambda r: r['financial_period.period']
+        year_sorted = sorted(results, key=year_key)
+        year_sorted.reverse()
+        quarters = {}
+        for year, yeargroup in groupby(year_sorted, year_key):
+            month_sorted =  sorted(yeargroup, key=month_key)
+            month_sorted.reverse()
+            # Loop over months that exist and add their values to quarters
+            for month, monthgroup in groupby(month_sorted, key=month_key):
+                monthitems = list(monthgroup)
+                quarter_idx = ((month - 1) / 3) + 1
+                quarter_key = "%sq%s" % (year, quarter_idx)
```

Comment:
could also use a tuple for this if we don't want to use this string in the frontend